hack/test-update-storage-objects.sh

```diff
@@ -93,7 +93,7 @@ startApiServer
 
 # Create a pod
 kube::log::status "Creating a pod"
-${KUBECTL} create -f examples/pod.yaml
+${KUBECTL} create -f docs/user-guide/pod.yaml
```

Comment:
It does:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/shippable.yml#L43

pod.yaml was copied to both directories previously.